libs/features/mas/mas/src/mas.js

```diff
@@ -1,26 +1,26 @@
 import { init } from '@adobe/mas-commerce';
-const { origin, searchParams } = new URL(import.meta.url);
+import '@adobe/mas-web-components/src/merch-card.js';
+import '@adobe/mas-web-components/src/merch-icon.js';
+import '@adobe/mas-web-components/src/merch-datasource.js';
 
+const { searchParams } = new URL(import.meta.url);
 const locale = searchParams.get('locale') ?? 'US_en';
-const lang = searchParams.get('lang') ?? 'en';
+const lang = searchParams.get('lang') ?? 'en'; // to be used in the future
 const isStage = searchParams.get('env') === 'stage';
-const features = searchParams.get('features');
 
 const envName = isStage ? 'stage' : 'prod';
 const commerceEnv = isStage ? 'STAGE' : 'PROD';
 
+const priceLiteralsPromise = fetch(
```

Comment:
but this logic (and URL) is already in merch.js, no? i guess we should factorize this

Reply:
this is needed for mas.js.  We can discuss about merch.js when we consider consuming mas.js in Milo.